### ISSUE TITLE
Underscore prepended to json file prevents pattern lab from generating

### ISSUE THREAD

**Issue by anthonysimone** [pattern-lab/patternlab-php#366]
In the recent update that allowed including patterns that were hidden by prepending `_` to the filename it seems that if the corresponding `json` file for that pattern is also prepended by `_` when you generate pattern lab you get an error.

If the `json` file is not prepended by `_` then it works as you expect. The error that I got when trying to generate is below. It wasn't all that helpful in tracking down the problem, so I just wanted to post here incase anyone else encountered the issue.

```
configuring pattern lab...
 wasn't found for loading. the given path: /Users/anthonysimone/Sites/yor/docroot/themes/custom/themekit/pattern-lab/source/_patterns/.twig


[09:23:12] 'pl:generate' errored after 495 ms
[09:23:12] Error: Command failed: /bin/sh -c php ./pattern-lab/core/console --generate

    at ChildProcess.exithandler (child_process.js:751:12)
    at ChildProcess.emit (events.js:110:17)
    at maybeClose (child_process.js:1015:16)
    at Process.ChildProcess._handle.onexit (child_process.js:1087:5)
```

**Comment by dmolsen** [comment]
@anthonysimone -

Ok, I'll create a better test case for this and see what the issue might be. Thanks for reporting it.

**Comment by dmolsen** [comment]
@anthonysimone -

Fixed in `dev` of core. Going to look at some other issues before cutting a release. I'll leave this open until that release goes out. Probably tonight. Thanks again.

**Comment by dmolsen** [comment]
This should have been fixed in `v2.7.0` of core. Please run `composer update` to get the latest version. I forgot to close this.

**Comment by dmolsen** [comment]
btw, here's a [full list of fixes](https://github.com/pattern-lab/patternlab-php-core/releases/tag/v2.7.0).
